### PR TITLE
Documentation for methods and usage

### PR DESCRIPTION
**Pull request by monde**

### CODE REVIEW

File: enroll.go [hunk 1]
```diff
@@ -163,21 +164,24 @@ func (r *EnrollmentResponse) ConfirmEmail(ctx context.Context, code string) (*En
...
+// OktaVeriftyOption constants.
 const (
 	OktaVerifyOptionQRCode OktaVerifyOption = "qrcode"
 	OktaVerifyOptionEmail  OktaVerifyOption = "email"
 	OktaVerifyOptionSms    OktaVerifyOption = "sms"
 )
 
 // VerifyPhone sends verification code to the provided phone.
-// Your phone number should contain a country code
+// Your phone number should contain a country code.
```
Comment (vijetmahabaleshwar-okta) [inline enroll.go:184]: nit: We also need to include the `+` sign in the phone number. Can we give an example of a number with that? Something like +11231231234?

Comment (monde) [reply enroll.go:184]: yes, I'll elaborate there.

File: reset_password.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ type Credentials struct {
 	Password string `json:"passcode"`
 }
```
Comment (vijetmahabaleshwar-okta) [inline reset_password.go:41]: Some of the methods below could use better comments, if you have time. :)

**Review by vijetmahabaleshwar-okta — APPROVED** [review]

File: reset_password.go (hunk 2 above)
Comment (monde) [reply reset_password.go:41]: lol, the methods names are self documenting but to write idiomatic/lint-able go code where all the public methods and functions are documented the comment tends to be redundant.

**Pull request merged**